modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Header/Header.scss

```diff
@@ -43,7 +54,15 @@
 			font-size: 18px;
 			font-weight: 600;
 			line-height: 150%;
+			max-width: 12.5rem;
+			overflow: hidden;
```

Comment:
@igor-franca `overflow` should be declared after `padding-right`.

Reply:
hey @naraandrade, I think you got confused here, can you check it please?

Reply:
Oh sorry, my mistake!